CHANGELOG.md

```diff
@@ -47,6 +48,7 @@ Notable changes to the framework should be documented here
 	  (ENT-7730)
 	- Reduced scope of report informing of missing systemd service
 	  (CFE-290, ENT-7360)
+	- Removed build dir from install/dist targets (ENT-7359)
```

Comment:
I imagine y'all may have removed this the last time around. That's fine.

Reply:
yeah this is fine to keep it affects how users use the source.